templates/cloudflare-workers/package.json

```
@@ -39,7 +38,7 @@
     "typescript": "^5.1.6",
     "vite": "^5.1.0",
     "vite-tsconfig-paths": "^4.2.1",
-    "wrangler": "3.57.1"
+    "wrangler": "^3.78.10"
```

In the past we've pinned this version because we've had unexpected breaking changes with `wrangler`

```suggestion
    "wrangler": "3.78.10"
```

---

Sorry to hear about the past issues with breaking changes! I'll make sure the team works to avoid this going forward. Happy to pin the version for now to get this moving. 👍🏼

---

Thanks Edmund! I appreciate that, and really appreciate you updating this template.